pkg/pipelinerunmetrics/metrics.go

```diff
@@ -0,0 +1,151 @@
+package pipelinerunmetrics
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/jonboulle/clockwork"
+
+	pipelinev1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"github.com/tektoncd/results/pkg/apis/config"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"go.uber.org/zap"
+	corev1 "k8s.io/api/core/v1"
+	"knative.dev/pkg/apis"
+	"knative.dev/pkg/metrics"
+)
+
+var (
+	prDeleteCount        = stats.Int64("results_pipelinerun_delete_count", "total number of deleted pipelineruns", stats.UnitDimensionless)
```

Comment:
Can we have `pipelinerun_delete_count` as the name? Do the need the `results` keyword, it's already implicit?

Reply:
Yes, just `pipelinerun_delete_count` is ok.  All metrics name that Results Watcher produce would prefixed with `watcher`.

Reply:
Modified.